test/lib/functionTester.js

```diff
@@ -0,0 +1,35 @@
+module.exports = createTestFunction;
+
+function createTestFunction( shouldBeCalled, times ){
+  if( times > 0 && !shouldBeCalled )
```

Comment:
In-place style for https://github.com/algolia/algoliasearch-client-js/blob/master/src/algoliasearch.js does not have spaces after parens. Pure styling issue but still, we must follow in-place style before we agree on how to style.

What do you think?